workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectSystemHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static class GameObjectSystemHelper
```

Comment:
I'd use the same name as the namespace. So `GameObjectRepresentation`. We seem to be inconsistent as to whether we add `SystemHelper` to each one.